CONTRIBUTE.md

```diff
@@ -0,0 +1,26 @@
+# Contribute
```

Comment:
That'd be great. I'll remove these then.
For the time being, can you share your setup please?

Reply:
The deps don't change often so you can just ignore those for now. Apart from that you can just mount or clone https://github.com/KurokuLabs/margo to `Packages/GoSublime/src/margo.sh` and almost everything should just work. I think only things like <kbd>ctrl+.</kbd>,<kbd>ctrl+p</kbd> won't work because they're being served by the older version of margo, but you should be able to work around that by manually setting the correct GOPATH in your project settings e.g. on Linux that'd be:

		"GoSublime": {
			"env": {
				"GOPATH": "$HOME/.config/sublime-text-3/Packages/User/GoSublime:$HOME/.config/sublime-text-3/Packages/GoSublime"
			},
		},